doc/nrf/device_guides/working_with_nrf/nrf70/developing/index.rst

```diff
@@ -9,11 +9,5 @@ The |NCS| provides support for development on the nRF70 Series, using the nRF700
    :maxdepth: 2
    :caption: Subpages:
 
-   powersave
    constrained
```

Comment:
Please see my comments in the SHEL ticket. Ideally this section should be split between the first part
- networking stack combinations
- host SoC support
- firmware patch optimizations
 that should stay under "Working with nRF70 Series", and a second part (the remaining parts) that should be moved under Protocols/Wi-Fi under a section named like "Wi-Fi stack configuration and performance" or similar